### DISCUSSION TITLE
Welcome to IGRP-Framework Discussions!

### DISCUSSION THREAD

**Discussion by NOSiopen** [NOSiCode-CV/IGRP-Framework#1752] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by poolborges** [comment]
**Paulo Borges (@poolborges)** 
- Fullstack developer for more than X years 
- Knowledge: JVM/Java, .NET/C#, PHP, C, Python, UML, SysML, Embedded System/MCU
- Interested in: Open Source, Software Architecture, Design Pattern, Security, Cloud, ..

**Can help in Framework**

- Design patterns
- Testing (Unit, Integration, ...)
- Code Quality (Formatting rules, Static Analysis, etc..) 
- Architecture, 

**Can help in "Go to Cloud"** 

- Architecture improvement for cloud
- Cloud Environment: Docker, Docker-compose (Development environment setup)
- GitOps (CI/CD) for Kubernetes
